.github/workflows/ci-cassandra.yml

```diff
@@ -47,12 +47,15 @@ jobs:
 
     - name: Setup CODECOV_TOKEN
       uses: ./.github/actions/setup-codecov
-
-    - name: Upload coverage to codecov
-      uses: codecov/codecov-action@894ff025c7b54547a9a2a1e9f228beae737ad3c2 # v3.1.3
+    
+    - name: Retry action for codecov uploads
```

Comment:
the name of the step is still `Upload coverage to codecov`

Reply:
sure